List.tsx

```diff
@@ -0,0 +1,28 @@
+import { useSelector } from "react-redux";
+import { IState } from "./store";
+import React from "react";
+import "./style.css";
+
+export const List: React.FC = () => {
+  const listOfUsers = useSelector((state: IState) => {
+    console.log(state);
+    return state?.listOfUsers;
+  });
+
+  React.useEffect(() => {

```

Comment:
Every line of code is executed and uses machine computational power
Don't add useless code

Reply:
Yes, I agree, I forgot to delete it.